src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorSupervisor.scala

```diff
@@ -63,11 +65,15 @@ class ClusterMonitorSupervisor(monitorConfig: MonitorConfig, dataprocConfig: Dat
       logger.info(s"Monitoring cluster ${cluster.projectNameString} for deletion.")
       startClusterMonitorActor(cluster, if (recreate) Some(RecreateCluster(cluster)) else None)
 
+    case ClusterUpdated(cluster) =>
+      logger.info(s"Monitor cluster ${cluster.projectNameString} for updating.")
+      startClusterMonitorActor(cluster, None)
```

Comment:
That's awesome the `ClusterMonitorActor` works as is without too many significant changes.